packages/office-ui-fabric-react/src/components/Panel/Panel.base.tsx

```diff
@@ -183,7 +183,6 @@ export class PanelBase extends BaseComponent<IPanelProps, IPanelState> implement
       <Layer {...layerProps}>
         <Popup
           role="dialog"
-          aria-modal="true"
```

Comment:
>ria-modal="true" [](start = 11, length = 16)

Is the panel the object or is it just a vehicle (I believe in this case it is the object)? If it is the object, I feel like this (or whatever element that's the actual Panel, not including the layer/overlay... just the container of the contents) is the one that should have the role and aria-modal